orderbook/src/database.rs

```diff
@@ -37,3 +41,34 @@ impl Database {
         Ok(())
     }
 }
+
+#[derive(sqlx::Type)]
+#[sqlx(rename = "OrderKind")]
+#[sqlx(rename_all = "lowercase")]
+enum DbOrderKind {
+    Buy,
+    Sell,
+}
+
+impl DbOrderKind {
+    fn from(order_kind: OrderKind) -> Self {
+        match order_kind {
+            OrderKind::Buy => Self::Buy,
+            OrderKind::Sell => Self::Sell,
+        }
+    }
+
+    fn into(self) -> OrderKind {
+        match self {
+            Self::Buy => OrderKind::Buy,
+            Self::Sell => OrderKind::Sell,
+        }
+    }
+}
+
+fn h160_from_vec(vec: Vec<u8>) -> Result<H160> {
```

Comment:
This has been moved to the conversion crate in my recent PR so we can expect conflicts at some point.